nitro.config.ts

```diff
@@ -8,7 +8,14 @@ export default defineNitroConfig({
         'access-control-allow-origin': '*',
       },
     },
-    '/mux': {
+    '/mux/upload': {
```

Comment:
You can specify these route rules on one line since they share a directory.

`/mux/**`